lib/chef/deprecation/warnings.rb

```diff
@@ -26,9 +26,8 @@ def add_deprecation_warnings_for(method_names)
           define_method(name) do |*args|
             message = []
             message << "Method '#{name}' of '#{self.class}' is deprecated. It will be removed in Chef 12."
```

Comment:
Sure, I just went through and fixed all instance of Chef 12 in the codebase (I'll submit here just for fun).